src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadContestedCaseDocumentsServiceTest.java

```diff
@@ -69,11 +90,12 @@ public void applicantCorrespondenceDocumentsFiltered() {
 
     @Test
     public void applicantFrFormsDocumentsFiltered() {
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form B", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Applicant - Form E", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form F", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form G", "applicant", "no", null));
-        uploadDocumentList.add(createContestedUploadDocumentItem("Form H", "applicant", "no", null));
+        when(featureToggleService.isRespondentJourneyEnabled()).thenReturn(false);
+        uploadDocumentList.add(createContestedUploadDocumentItem("Form B", "applicant", "no", null, null));
```

Comment:
Added new 'null' parameter to all old tests to reflect the addition of "hearingDetails' text field